app/1.0/docs/devguide/data-binding.md

```diff
@@ -4,39 +4,71 @@ title: Data binding
 
 <!-- toc -->
 
-Data binding binds a property or sub-property of a custom element (the _host element_) to
-a property or attribute of an element in its local DOM (the _child_ or _target element_).
+A _data binding_ connects data from a custom element
+(the _host element_) to a property or attribute of an element in its local DOM (the _child_
+or _target element_). The host element data can be a property or sub-property represented by a
+[data path](data-system#data-paths), or data generated based on one or more paths.
 
-A binding is created with a binding _annotation_ in the host element's local DOM template:
+You create data bindings by adding annotations to an element's local DOM template.
 
 ```
 <dom-module id="host-element">
     <template>
```

Comment:
indentation seems off. 4 spaces for <template>, then 2 spaces for <target-element>